### PR TITLE
Start Relay Resolver Docs with basic examples

### PR DESCRIPTION
**Pull request by youngkidwarrior**
Created these based off the part of resolvers I have used. I tried to take a stab at graphql fragment arguments, but I have never had to use those so not sure if correct

I just used the `UserGreeting` example the relay docs used. We might want a more "rescripty" example that shows off how well resolvers work with Variants and switch statement

TODO:
- [ ] Check graphql argument code and make sure it is correct
- [ ] Write docs passing runtime arguments

### CODE REVIEW

File: rescript-relay-documentation/docs/relay-resolvers.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+## Basic Example
+```js
+// UserGreetingResolver
+/**
+ * @RelayResolver
+ *
+ * @onType User
+ * @fieldName greeting
+ * @rootFragment GreetingResolver
```
Comment (zth) [inline rescript-relay-documentation/docs/relay-resolvers.md:45]: I think there's a more compact notation now that we could perhaps also show. Can't find a link to it now though.

**Review by zth — APPROVED** [review]
Looking great!

**Comment by zth** [comment]
I haven't tried passing JS arguments to resolvers in RescriptRelay yet myself. I would assume we need some deeper integration for that to work.

**Comment by youngkidwarrior** [comment]
Okay, I'll take that part out for now

**Comment by zth** [comment]
Superseded by recent doc updates. Thanks for the effort anyway!

**Pull request closed without merging**